CMakeLists.txt

```diff
@@ -563,6 +563,11 @@ if(openPMD_HAVE_ADIOS1)
         set_target_properties(openPMD.ADIOS1.Serial PROPERTIES
             LINK_FLAGS "-Wl,--exclude-libs,ALL")
     elseif("${CMAKE_SYSTEM_NAME}" MATCHES "Darwin")
+        check_linker_flag(CXX "-Wl,--exclude-libs,ALL" DARWIN_WO_XCODE)
```

Comment:
This might be a CMake 3.18+ feature:
- https://gitlab.kitware.com/cmake/cmake/-/merge_requests/4777
- https://cmake.org/cmake/help/latest/module/CheckLinkerFlag.html

Thus:
- [ ] need to bump CMake dependency from 3.15+ to 3.18+

Reply:
Also: should use `LINKER:`:
Expands to `-Wl,--...` on GCC and to `-Xlinker --...` on Clang.